### ISSUE TITLE
Remove retailer and wholesaler option on 'List a Venue' page

### ISSUE THREAD

**Issue by jessyclubsoda** [club-soda/club-soda-guide#655]
We have recently had a couple of venues who also might operate as shops try and list themselves as 'retailers' when they are listing. However, this means they are not publicly viewable and instead get pushed into the back end where retailers can be manually updated with their drinks listed by admins.

As we mainly use retailers to refer to online retailers, and monitor that as admins, we would like to remove the option for venues to list themselves as either retailers or wholesalers

**Comment by iteles** [comment]
@jessyclubsoda Thanks for opening this issue. We would still want to keep this as an option for admins, but just no longer give venues themselves that option?

**Comment by jessyclubsoda** [comment]
@iteles 

Yes - we **definitely** want it removed from this public version of the 'List a Venue' page
![Screenshot 2020-04-15 09 56 52](https://user-images.githubusercontent.com/49029914/79318903-8ae14200-7eff-11ea-8f45-7db50a88c357.png)

We wouldn't even need it to appear on any admin version of Listing a Venue either because Retailers and Wholesalers can and should just be added by admins through their relevant URLs www.clubsodaguide.com/admin/retailers and www.clubsodaguide.com/admin/wholesalers
